packages/trip-form/src/ModeIcon/index.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import PropTypes from "prop-types";
+import * as Icons from "@opentripplanner/icons";
+
+/**
+ * A generic icon component that displays an icon for the specified transportation mode.
+ */
```

Comment:
Is this duplicative of https://github.com/opentripplanner/otp-ui/blob/master/packages/icons/src/trimet-mode-icon.js?

Reply:
I didn't see that one... the original code I based this work was referring to this file instead: https://github.com/opentripplanner/otp-react-redux/blob/dev/lib/components/icons/mode-icon.js

If creating an issue, is it against this repo or against otp-react-redux?

Reply:
Probably this repo is OK.